docs/0.4/concepts/security_considerations.md

```diff
@@ -0,0 +1,164 @@
+# Security Considerations
+
+<!--
+  Copyright 2018-2020 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+Splinter provides several levels of security and customizable permissions.
+
+## Cryptographic signing with public/private keys
+
+Like Sawtooth and other distributed ledgers, Splinter uses cryptographic
+signing with public/private keys. Each user and transaction-submitting entity
+has a public key and an encrypted private key.
+
+All transactions that affect shared state are signed with the encrypted private
```

Comment:
Remove "encrypted" here, as it doesn't make sense in this context (signing is done with the unencrypted key, but this isn't really about how its stored). (Also remove encrypted in the previous paragraph.) Also, this should probably be "a private key" not "the private key".

Reply:
Removed "encrypted" (private key) while rewriting this section.